Sources/FoundationEssentials/FileManager/FileManager+Basics.swift

```diff
@@ -16,6 +16,28 @@ import Darwin
 import Glibc
 #elseif os(Windows)
 import CRT
+import WinSDK
+#endif
+
+#if os(Windows)
+extension _FILE_ID_128: Equatable {
+    public static func == (_ lhs: _FILE_ID_128, _ rhs: _FILE_ID_128) -> Bool {
+        return withUnsafePointer(to: lhs.Identifier.0) { pLHS in
```

Comment:
Is `lhs.Identifier` a tuple of size 16? If so I think you'll likely need `withUnsafePointer(to: lhs.Identifier)` instead of the pointer to `.0` - `withUnsafePointer` only guarantees you a pointer to what you provide it with so getting a pointer to the first element in the tuple doesn't guarantee that the remaining elements will follow, but getting a pointer to the tuple itself should work fine

Reply:
Yes, its a tuple of size 16. Okay, I'll change that to `lhs.Identifier` then.